lms/djangoapps/ccx/tests/utils.py

```diff
@@ -8,14 +8,15 @@
 
 from lms.djangoapps.ccx.overrides import override_field_for_ccx
 from lms.djangoapps.ccx.tests.factories import CcxFactory
+from openedx.core.djangoapps.ace_common.tests.test_templatetags import EmailTemplateTagMixin
```

Comment:
We should move EmailTemplateTagMixin to mixins.py in that directory, rather than importing from a test file.